strawberryfields/apps/qchem/dynamics.py

```diff
@@ -250,3 +254,84 @@ def prob(samples: list, excited_state: list) -> float:
         raise ValueError("The excited state must not contain negative values")
 
     return samples.count(excited_state) / len(samples)
+
+
+def sample_tmsv(
+    sq: list, t: float, Ul: np.ndarray, w: np.ndarray, n_samples: int, loss: float = 0.0,
+) -> list:
+    r"""Generate samples for simulating vibrational quantum dynamics with a two-mode squeezed input state.
+
+    Given :math:`N` squeezing parameters and :math:`N`-dim normal-to-local transformation matrix,
+    a GBS device with :math:`2N` modes is simulated. The evolution operator acts on only the first
```

Comment:
change to: "this function generates samples from a GBS device with two-mode squeezed vacuum input states."

Reply:
As the beginning of the second sentence in the doc string?